cryptid_anchor/client/packages/idl/cryptid_anchor.ts

```diff
@@ -465,58 +476,53 @@ export type CryptidAnchor = {
     },
     {
       "code": 6003,
+      "name": "WrongCryptidAccount",
```

Comment:
I guess in after-live releases we need to make sure that errors are only added to the end :)